sheoranharis.json

```
@@ -0,0 +1,23 @@
+{
+    "name": "Harish Sheoran",
+    "type": "personal",
+    "bio": "Pythonista",
+    "avatar": "https://github.com/sheoranharis.png",
+    "links": [
+        {
+            "name": "Follow me on GitHub",
+            "url": "https://github.com/sheoranharis",
+            "icon": "github"
+        },
+        {
+            "name": "Follow me on Twitter",
+            "url": "https://twitter.com/sheoranharis",
+            "icon": "twitter"
+        }
+      {
+            "name": "Read my Blog ",
```

Extra space here. Might want to remove it. 🙂
```suggestion
            "name": "Read my Blog",
```